sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/CosmosContainer.java

```diff
@@ -246,21 +280,61 @@ CosmosItemResponse mapItemResponseAndBlock(Mono<CosmosAsyncItemResponse> itemMon
      * @return the iterator
      */
     public Iterator<FeedResponse<CosmosItemProperties>> queryChangeFeedItems(ChangeFeedOptions changeFeedOptions) {
-        return getFeedIterator(this.containerWrapper.queryChangeFeedItems(changeFeedOptions));
+        return getFeedIterator(this.asyncContainer.queryChangeFeedItems(changeFeedOptions));
     }
 
     /**
-     * Gets item.
+     * Read cosmos sync item response.
      *
-     * @param id the id
+     * @param itemId the item id
      * @param partitionKey the partition key
-     * @return the item
+     * @return the cosmos sync item response
+     * @throws CosmosClientException the cosmos client exception
+     */
+    public <T> CosmosItemResponse<T> readItem(String itemId, PartitionKey partitionKey, Class<T> itemType) throws CosmosClientException {
+        return this.mapItemResponseAndBlock(asyncContainer.readItem(itemId,
+                                                                    partitionKey,
+                                                                    new CosmosItemRequestOptions(),
+                                                                    itemType));
+    }
+
+    /**
+     * Read cosmos sync item response.
+     *
+     * @param options the options
+     * @return the cosmos sync item response
+     * @throws CosmosClientException the cosmos client exception
+     */
+    public <T> CosmosItemResponse<T> readItem(String itemId, PartitionKey partitionKey,
+                                          CosmosItemRequestOptions options, Class<T> itemType) throws CosmosClientException {
+        return this.mapItemResponseAndBlock(asyncContainer.readItem(itemId, partitionKey, options, itemType));
+    }
+
+    /**
+     * Replace cosmos sync item response.
+     *
+     * @param item the item
+     * @param options the options
+     * @return the cosmos sync item response
```

Comment:
Please fill the gaps of missing`@param` here and other API docs.

Reply:
Done